lib/wiki_course_edits.rb

```diff
@@ -12,17 +13,20 @@ def initialize(action:, course:, current_user:, **opts)
     @course = course
     # Edits can only be made to the course's home wiki through WikiCourseEdits
     @home_wiki = course.home_wiki
+    return unless @home_wiki.edits_enabled?
     @wiki_editor = WikiEdits.new(@home_wiki)
     @dashboard_url = ENV['dashboard_url']
     @current_user = current_user
+    template_file_path = "config/templates/#{@dashboard_url}_#{@home_wiki.language}.yml"
+    @templates = YAML.load_file(Rails.root + template_file_path)
```

Comment:
Pass @templates to WikiAssignmentOutput and WikiCourseOutput, instead of redoing this in each file.